src/main.js

```diff
@@ -129,6 +129,16 @@ const initSettings = () => {
   };
 };
 
+pluginLoader.register('@ima/plugin-atoms', (ns) => {
+  $registerImaPlugin(ns);
+
+  return {
+    initBind,
+    initServices,
+    initSettings,
```

Comment:
All of the 3 functions above should probably not be exported from the package. Or if this should be backwards compatible change, the `pluginLoader` should be wrapped in `if (typeof pluginLoader !== 'undefined')` statement.

Also you could probably even inline those 3 functions directly into the register call but I will leave that up to you.